topi/python/topi/arm_cpu/conv2d.py

```diff
@@ -119,7 +123,11 @@ def _callback(op):
             if isinstance(kernel.op, tvm.tensor.ComputeOp) and "dilate" in kernel.op.tag:
                 s[kernel].compute_inline()
 
-            _schedule_spatial_pack(cfg, s, data_vec, kernel_vec, conv, output, outs[0])
+        # TODO: move to schedule_nhwc later
+            if 'nhwc' in op.tag:
```

Comment:
this is in the `schedule_conv2d_nchw_arm_cpu` function - should 'nhwc' ever be in op.tag?

Reply:
reverted changes to arm nhwc schedule in favor of jackwish version.